src/NServiceBus.NHibernate/NServiceBus.NHibernate.csproj

```diff
@@ -15,7 +15,8 @@
     <SignAssembly>true</SignAssembly>
     <TargetFrameworkProfile />
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <NuGetPackageImportStamp>ec59a453</NuGetPackageImportStamp>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
My idea was that it would be a one off, but if vs2015 just adds it back lets leave it